plugins/cloudinary/src/CloudinaryCredentialsDialog.tsx

```diff
@@ -0,0 +1,82 @@
+/** @jsx jsx */
+import { jsx } from '@emotion/core'
+import React from 'react'
+import {
+  Dialog,
+  Button,
+  DialogActions,
+  DialogTitle,
+  DialogContent,
+  TextField
+} from '@material-ui/core'
+
+interface CloudinaryCredentialsDialogProps {
+  openDialog: boolean
+  closeDialog(): void
+  updateCloudinaryCredentials(apiKey: string, cloudName: string): void
+}
+
+interface CloudinaryCredentialsDialogState {
+  apiKey: string
+  cloudName: string
+}
+
+export default class CloudinaryCredentialsDialog extends React.Component<
+  CloudinaryCredentialsDialogProps,
+  CloudinaryCredentialsDialogState
+> {
+  state = {
+    apiKey: '',
+    cloudName: ''
```

Comment:
it would be great to set these to their current values if already set, that way on subsequent opens it won't confuse the user into thinking they haven't set these if they display blank but the settings do actually already exist

Reply:
ok!